locales/it/index.yml

```
@@ -3189,7 +3189,7 @@ features:
       eid: Identità Digitale
       mdl: Patente di Guida
       dc: Carta Europea della Disabilità
-      ts: Tessera Sanitaria
+      ts: Tessera Sanitaria - Tessera europea di assicurazione malattia
```

This is used for the name of a credential in the cards within the wallet. If we need a different copy for the screen title we should add new keys for it.

```suggestion
      ts: Tessera Sanitaria
```

---

I updated the cards assets in 0d9ac0b30cd4367e64d1eb8e36e334067b4fa5b4.